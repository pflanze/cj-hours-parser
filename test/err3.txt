set foo 1
foo = 2
set foo<> = 2
foo<> = 2
