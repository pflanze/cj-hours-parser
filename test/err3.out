item=ProcessingInstruction { key: "foo", value: "1" }
    =ProcessingInstruction { key: "foo", value: "1" }
item=ProcessingInstruction { key: "foo", value: "2" }
    =ProcessingInstruction { key: "foo", value: "2" }
